src/main/java/uk/gov/hmcts/reform/civil/handler/tasks/GenerateCsvAndTransferTaskHandler.java

```diff
@@ -98,4 +99,12 @@ private String generateCSVRow(String[] row) {
         return builder.toString();
     }
 
+    private String[] getCSVHeaders() {
+        if (toggleService.isLipVLipEnabled()) {
+            return new String[] {"SITE_ID", "CASE_NUMBER", "CASE_TYPE", "AMOUNT", "PARTY_TYPE", "COMPANY_NAME", "CONTACT_NAME",
```

Comment:
is it possible to create common array/list and welsh flag to that if isLipVLipEnabled.